src/reducers/Setting.js

```diff
@@ -1,13 +1,20 @@
 import {
+  DEFAULT_LANGUAGE,
   HEADER_BACKGROUND_COLOR,
 } from '../types';
 
 const INITIAL_STATE = {
+  defaultLanguage: 'en',
   headerBackgroundColor: '#2196F5',
 };
 
 export default (state = INITIAL_STATE, { type, payload }) => {
   switch (type) {
+    case DEFAULT_LANGUAGE:
```

Comment:
Your cases should be more descriptive. And better if they are verbs. Example: `SET_LANGUAGE`, `SET_HEADER_BACKGROUND_COLOR`, etc.